dependencies.props

```diff
@@ -30,6 +31,7 @@
     <ExternalExpectedPrerelease>beta-24727-00</ExternalExpectedPrerelease>
     <ProjectNTfsExpectedPrerelease>beta-24926-00</ProjectNTfsExpectedPrerelease>
     <StandardExpectedPrerelease>beta-24919-01</StandardExpectedPrerelease>
+    <SniExpectedPrerelease>beta-24926-02</SniExpectedPrerelease>
```

Comment:
Is "beta" the pre-release prefix being used by the runtime packages generated in your repo?